src/popup/router/index.ts

```diff
@@ -70,10 +70,19 @@ router.beforeEach(async (to, from, next) => {
     return;
   }
 
-  if (to.name === ROUTE_APPS_BROWSER && activeAccount.value.protocol !== PROTOCOL_AETERNITY) {
-    setActiveAccountByIdx(0);
-    next({ name: ROUTE_APPS_BROWSER });
-    return;
+  if (to.name === ROUTE_APPS_BROWSER) {
+    // In-app browser is mobile-only
+    if (!IS_CORDOVA) {
+      next({ name: ROUTE_INDEX });
+      return;
+    }
+
+    // In-app browser only works with AE accounts
+    if (activeAccount.value.protocol !== PROTOCOL_AETERNITY) {
```

Comment:
Some similar logic was already done in https://github.com/aeternity/superhero-wallet/pull/2264, please review it and align both of your PRs to the common ground.

Reply:
This check is not relevant to the mobile-only PR. 
But we should update the #2242 PR once #2264 is merged